pages/[slug]/index.js

```diff
@@ -44,6 +47,28 @@ const Page = ({ slug, data, html, locales }) => {
       >
         <Authors text={data.author} />
       </Header>
+
+      <Container
+        sx={{
+          bottom: '25px',
+          position: 'fixed',
+          right: '0',
+          width: 'auto'
+        }}
+      >
+        <Button
+          as="a"
+          variant="outline"
+          href={`https://github.com/hackclub/hackclub/edit/main/workshops/${slug}/README.md`}
+          sx={{
+            backgroundColor: 'background',
+          }}
+        >
+          <Edit3 />
+          Edit this page!
```

Comment:
probs wrap this in a `<Text>` component with `sx={{//the stuff I sent early}}`

Reply:
@sampoder Perfect, that's working great, thank you! Your instructions were very helpful 🙏 

Last thing I'm messing with is that there's a margin on the edit SVG that i'm trying to get rid of if the text isn't there, but it doesn't seem to be applying the styles from `sx`. I'm new to `theme-ui` so I'm clunking through 😅, thanks for your help!

Reply:
Ahh I think that's a react feather icon which is separate... I'll look into it

Reply:
Could you send me a screenshot of the margin?

Reply:
Here you go! Specifically it's the `margin-right` we want to set to `0`.

<img width="486" alt="Screen Shot 2022-04-15 at 11 45 55 AM" src="https://user-images.githubusercontent.com/621904/163591656-b44ce459-6710-481e-8379-d8c82c9ef1b0.png">

Reply:
That's set by the theme: https://github.com/hackclub/theme/blob/95a382891132241dc3a517d091554da1e482e339/packages/theme/src/index.ts#L215

Reply:
oh yeah good catch!